addons/survey/static/src/js/survey_form.js

```diff
@@ -537,6 +549,77 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
         }
     },
 
+    _prepareNextScreen: function(submitPromise, options) {
+        // Make the form completely readonly
+        const form = document.querySelector('form');
+        form.querySelectorAll('input, textarea, label, td')?.forEach(node => {
+            node.classList.add("pe-none");
+        });
+        // Replace the Submit button by a Next button
+        form.querySelector('button[type="submit"]')?.classList.add('d-none');
+        const nextPageBtn = form.querySelector('button[id="next_page"]');
+        nextPageBtn?.classList.remove('d-none');
+        nextPageBtn?.addEventListener('click', () => {
+            this._nextScreen(submitPromise, options);
+        });
+        // On enter or arrow right go to next page
+        const nextPageKeydownListener = (event) => {
+            if (event.code === 'Enter' || event.code === 'ArrowRight') {
```

Comment:
And in the other keys cases (are you are replacing `onKeyDown` which does "return" too. If we do want to block all that, we should probably add a comment.